www-servers/pound/pound-4.16.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# TODO: Add adns as a dependency in order to allow building with support for
+#       dynamic backends
+# TODO: Add hoard as a dependency in order to support the --enable-hoard configure
```

Comment:
I would explicitly disable both of these via configure args to avoid automagic from overlays.